c10/core/impl/PyObjectSlot.h

```diff
@@ -24,11 +25,9 @@ struct C10_API PyObjectSlot {
   //
   // NB: THIS FUNCTION CAN RAISE AN EXCEPTION.  Make sure to clean up after
   // PyObject if necessary!
-  void init_pyobj(
-      PyInterpreter* self_interpreter,
-      PyObject* pyobj,
-      PyInterpreterStatus status) {
-    pyobj_interpreter_.store(self_interpreter, std::memory_order_relaxed);
+  void init_pyobj(PyObject* pyobj) {
+    pyobj_interpreter_.store(
```

Comment:
We are going to remove this field in a follow up right?
It will be a nice struct size win :)